third_party/move/move-compiler-v2/tests/control-flow-simplification/jump-label.move

```diff
@@ -0,0 +1,67 @@
+module 0x42::test {
```

Comment:
Your PR description says:

> When there is block goto L followed by block label L where the latter has only one predecessor, we may generate an empty block in the control flow graph, an invariant violation. We fixed this bug and also introduced some more invariant checks.

This test case doesn't seem to crash when I add it to the current main branch.  Do we have a test to illustrate the motivating bug?

Can you link to an issue?

Reply:
I tried again and it crashes with
```
thread 'main' panicked at third_party/move/move-compiler-v2/tests/testsuite.rs:915:9:
cannot find test configuration for `tests/control-flow-simplification/bug-10253.move`
```
Maybe you didn't turn on CFG simplification? You can add the test case to say `third_party/move/move-compiler-v2/tests/file-format-generator` to reproduce.

We haven't opened an issue for this.

Reply:
That's a crash just because testsuite.rs doesn't know about tests/control-flow-simplification/.  You also need to add the testsuite.rs change from this PR.

Reply:
I'd still like to understand the bug.  Using your changes only under move-compiler-v2/tests/ I see no crash, and in fact identical output, which is good to show this didn't break anything, but I don't see what it's fixing.

Reply:
That was a huge oversight... I now updated the test, and it should crash on main if you put it under `third_party/move/move-compiler-v2/tests/file-format-generator`